devicetypes/smartthings/nyce-open-closed-sensor.src/nyce-open-closed-sensor.groovy

```diff
@@ -353,9 +356,8 @@ Integer convertHexToInt(hex) {
 }
 
 def refresh() {
-	log.debug "Refreshing Battery"
 	def refreshCmds = [
-			"st rattr 0x${device.deviceNetworkId} ${endpointId} 1 0x20", "delay 200"
+		"st rattr 0x${device.deviceNetworkId} ${endpointId} 1 0x20", "delay 200"
```

Comment:
Could you also update the DTH to use the `zigbee` library wherever possible, for example replace this with `zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0020)`

Reply:
Sure,  Tom. I will work on it.

Reply:
I need so more work not to let device to be disconnected.